cdap-ui/app/cdap/components/Cloud/Profiles/DetailView/Content/BasicInfo/index.js

```diff
@@ -0,0 +1,198 @@
+            extendedDeleteErrMsg: err,
+            deleteLoading: false
+          });
+        });
+  };
+
+  renderDeleteConfirmationModal() {
+    if (!this.state.deleteModalOpen) {
+      return null;
+    }
+
+    const confirmationText = T.translate(`${PREFIX}.common.deleteConfirmation`, {profile: this.props.profile.name});
+
+    return (
+      <ConfirmationModal
+        headerTitle={T.translate(`${PREFIX}.common.deleteTitle`)}
+        toggleModal={this.toggleDeleteModal}
+        confirmationText={confirmationText}
+        confirmButtonText={T.translate('commons.delete')}
+        confirmFn={this.deleteProfile}
+        cancelFn={this.toggleDeleteModal}
+        isOpen={this.state.deleteModalOpen}
+        errorMessage={this.state.deleteErrMsg}
+        extendedMessage={this.state.extendedDeleteErrMsg}
+        isLoading={this.state.deleteLoading}
+      />
+    );
+  }
+
+  renderProfileInfoGrid() {
+    let profile = this.props.profile;
+
+    return (
+      <div className="grid-wrapper profile-info-grid">
+        <div className="grid grid-container">
+          <div className="grid-header">
+            <div className="grid-row">
+              <strong>{T.translate(`${PREFIX}.common.provisioner`)}</strong>
+              <strong>{T.translate('commons.scope')}</strong>
+              <strong>{T.translate(`${PREFIX}.common.last24HrRuns`)}</strong>
+              <strong>{T.translate(`${PREFIX}.common.totalRuns`)}</strong>
+              <strong>{T.translate(`${PREFIX}.common.last24HrNodeHr`)}</strong>
+              <strong>{T.translate(`${PREFIX}.common.totalNodeHr`)}</strong>
+              <strong>{T.translate(`${PREFIX}.DetailView.creation`)}</strong>
+            </div>
+          </div>
+          <div className="grid-body">
+            <div className="grid-row">
+              <div>
+                <IconSVG name="icon-cloud" />
+                <span>{profile.provisioner.name}</span>
+              </div>
+              <div>{profile.scope}</div>
+              <div />
+              <div />
+              <div />
+              <div />
+              <div />
+            </div>
+          </div>
+        </div>
+      </div>
+    );
+  }
+
+  render() {
+    let profile = this.props.profile;
+    let redirectToObj = this.props.isSystem ? {
+      pathname: '/administration/configuration',
+      state: { accordionToExpand: ADMIN_CONFIG_ACCORDIONS.systemProfiles }
+    } : `/ns/${getCurrentNamespace()}/details`;
+    let namespace = this.props.isSystem ? 'system' : getCurrentNamespace();
+
+    const actionsElem = () => {
+      return (
+        <div>
+          <IconSVG name="icon-cog-empty" />
+          <div>Actions</div>
+        </div>
+      );
+    };
+
+    return (
+      <div className="detail-view-basic-info">
+        <div className="profile-name-delete">
+          <h2 className="profile-name">
+            {profile.name}
+          </h2>
+          <Popover
```

Comment:
Could we make a generic `ActionsPopover` for all the profile views? We seem to be having this list view and now in detail view and their behavior seems to be the same. They behave differently on different profile ids.